stdlib/CompilerSupportLibraries_jll/src/CompilerSupportLibraries_jll.jl

```diff
@@ -57,7 +57,7 @@ function __init__()
     global libgomp_handle = dlopen(libgomp)
     global libgomp_path = dlpath(libgomp_handle)
     global artifact_dir = dirname(Sys.BINDIR)
-    global LIBPATH[] = dirname(libgcc_s_path)
+    LIBPATH[] = dirname(libgcc_s_path)
```

Comment:
Just out of curiosity, why is this necessary (it is not mentioned in the PR description nor the commit message, as far as I could tell)

Reply:
It is just a redundant `global` annotation. It doesn't do anything.